### ISSUE TITLE
problem with plots count

### ISSUE THREAD

**Issue by bartassw** [Chia-Network/chia-blockchain#3513]
I have a problem with plots count.

When i run chia plot check - i have information about 550 plots, but when i start farming summary: 

Farming status: Farming
Total chia farmed: 6.0
User transaction fees: 0.0
Block rewards: 6.0
Last height farmed: 14xxx
Plot count: 0
Total size of plots: 0.000 GiB
Estimated network space: 1752.681 PiB
Expected time to win: Unknown

this hapend after harvester restart - i decide to restart it, because it was showing me only 440 plots.... 

is my harvester working properly?
maybe i have som wrong config?
this computer previously it worked as a harvester, but due to the fact that I had problems with a computer that was full_mode under windows, I ran other services on this harvester and turned it off on windows.

please help me.

**Comment by gbrown2036** [comment]
I'm having a similar issue with "Plot count: 0" even though "chia plots check" returns 15 plots.  Did you figure out how to resolve this?

**Comment by x1957** [comment]
same problem

**Comment by karolcichosz** [comment]
Same story. chia plots check returns 10 plots. chia farm summary 0.

**Comment by karolcichosz** [comment]
> Same story. chia plots check returns 10 plots. chia farm summary 0.

Update: After chia services restart the farming part has started to see these 10 plots, but in general it should be automatic, so I consider it as a potential BUG to check.

**Comment by SlavikCA** [comment]
Same problem here.

Logs (keys removed):
```
slavik@ds1817:/volume1/hosting/piGallery2$ sudo docker exec -it chia venv/bin/chia farm summary
Farming status: Syncing
Total chia farmed: 0.0
User transaction fees: 0.0
Block rewards: 0.0
Last height farmed: 0
Plot count: 0
Total size of plots: 0.000 GiB
Estimated network space: 5049.219 PiB
Expected time to win: Unknown

slavik@ds1817:/volume1/hosting/piGallery2$ sudo docker exec -it chia venv/bin/chia plots check
2021-05-16T20:15:38.659  chia.plotting.check_plots        : INFO     Loading plots in config.yaml using plot_tools loading code

2021-05-16T20:15:41.880  chia.plotting.plot_tools         : INFO     Searching directories ['/plots']
2021-05-16T20:15:42.445  chia.plotting.plot_tools         : INFO     Found plot /plots/plot-k32-2021-05-15-03-30-aea315e094cb3cd2629a0386e9e291dddf9862449869ee3548b55f7720b87e60.plot of size 32
2021-05-16T20:15:42.603  chia.plotting.plot_tools         : INFO     Found plot /plots/plot-k32-2021-05-13-22-46-2707a4cc72a6bfc8cc619d68e69c9bcbd949150f3e0ccda8736d5e4f63689fa6.plot of size 32
2021-05-16T20:15:42.607  chia.plotting.plot_tools         : INFO     Found plot /plots/plot-k32-2021-05-15-05-04-8ca6fb5f005d49cfc31fc8c37b5a2f8b505aa767a0149f271b585af9f275098e.plot of size 32
2021-05-16T20:15:42.637  chia.plotting.plot_tools         : INFO     Found plot /plots/plot-k32-2021-05-14-05-55-b93e2c5775cf0165888de20808718b96324f763c3853294d76ab57075fbd3e8d.plot of size 32
2021-05-16T20:15:42.638  chia.plotting.plot_tools         : INFO     Loaded a total of 4 plots of size 0.3958817960046872 TiB, in 0.8756387233734131 seconds
2021-05-16T20:15:42.639  chia.plotting.check_plots        : INFO
2021-05-16T20:15:42.639  chia.plotting.check_plots        : INFO
2021-05-16T20:15:42.640  chia.plotting.check_plots        : INFO     Starting to test each plot with 30 challenges each

2021-05-16T20:15:42.641  chia.plotting.check_plots        : INFO     Testing plot /plots/plot-k32-2021-05-15-03-30-aea315e094cb3cd2629a0386e9e291dddf9862449869ee3548b55f7720b87e60.plot k=32
2021-05-16T20:16:21.189  chia.plotting.check_plots        : INFO        Proofs 32 / 30, 1.0667
2021-05-16T20:16:21.190  chia.plotting.check_plots        : INFO     Testing plot /plots/plot-k32-2021-05-13-22-46-2707a4cc72a6bfc8cc619d68e69c9bcbd949150f3e0ccda8736d5e4f63689fa6.plot k=32
2021-05-16T20:16:51.916  chia.plotting.check_plots        : INFO        Proofs 31 / 30, 1.0333
2021-05-16T20:16:51.917  chia.plotting.check_plots        : INFO     Testing plot /plots/plot-k32-2021-05-15-05-04-8ca6fb5f005d49cfc31fc8c37b5a2f8b505aa767a0149f271b585af9f275098e.plot k=32
2021-05-16T20:17:31.972  chia.plotting.check_plots        : INFO        Proofs 49 / 30, 1.6333
2021-05-16T20:17:31.972  chia.plotting.check_plots        : INFO     Testing plot /plots/plot-k32-2021-05-14-05-55-b93e2c5775cf0165888de20808718b96324f763c3853294d76ab57075fbd3e8d.plot k=32
2021-05-16T20:18:12.723  chia.plotting.check_plots        : INFO        Proofs 36 / 30, 1.2
2021-05-16T20:18:12.724  chia.plotting.check_plots        : INFO
2021-05-16T20:18:12.724  chia.plotting.check_plots        : INFO
2021-05-16T20:18:12.724  chia.plotting.check_plots        : INFO     Summary
2021-05-16T20:18:12.725  chia.plotting.check_plots        : INFO     Found 4 valid plots, total size 0.39588 TiB
2021-05-16T20:18:12.725  chia.plotting.check_plots        : INFO     4 plots of size 32
```

**Comment by Trustmeimtheninja** [comment]
I'm having this same problem. I have 100+ plots that show during the chia plots check, but they don't show when I ask for the summary. I don't want to restart my Chia, because I'm just now getting good sync times. Starting it over kinda scares me. And needing to do this every day isn't my idea of a good time.

**Comment by ravensorb** [comment]
Same issue.  There doesn't seem to  be a way to tell the farmer to see the new plots without restarting.  Is there a hidden CLI command?  I think there might be as the GUI is able to trigger a "refresh"...

**Comment by room101-dev** [comment]
Easy to duplicate problem, been ongoing for months, this is the primary reason most have moved to hpool, because the 'farmer' just does't work, while the blockchain appears to be syncing, and the wallet, and you can 'chia plots show' and see the plots and 'chia plots check' and check the plots, when you do a 'chia farm summary', you get zero's all the way down the list.

This is a bug, but nobody in dev uses linux, so they don't care, its obviously not a problem oh a mac or windows, as you don't see anybody complaining.

I quit depending upon the ability of chia-dev to have their 'farmer' running months ago, I gave up and moved over to hpool, but I still test once a week to see if stuff is working, and of course its still not working.

**Comment by PsyMan2000** [comment]
getting this on windows now, 155 plots on check, none on farm. taken a week to make those and looks like I will have to delete and start over, worth the electricity to plat them all again? maybe not, just glad I have not burned through multiple NVMe drives plotting something that appears to be pretty fragile longer term.

**Comment by Trustmeimtheninja** [comment]
> getting this on windows now, 155 plots on check, none on farm. taken a week to make those and looks like I will have to delete and start over, worth the electricity to plat them all again? maybe not, just glad I have not burned through multiple NVMe drives plotting something that appears to be pretty fragile longer term.

I backed up my db files, deleted them from the original location, started the farmer, stopped the farmer, killed all chia processes, put the backup of the db files back into the original location, started the farmer again and it all synced with my plots working again. It's been a bit since I did this. Let me know if the process makes sense.

**Comment by PsyMan2000** [comment]
Brilliant, I backed up the db folder in main net to save a full resync, uninstalled, deleted the .chia folder and reinstalled, stopped, replaced the db folder and run it again. It must have retained my keys in the registry as it did not ask me to create new ones and I could log in but I did have to add the plot folders again and BAM, farming 156 plots again. Thanks for the tips, they aren't quite as fragile as I feared. Good work The Ninja :-D
